### PR TITLE
the LORE was WRONG

### PR DESCRIPTION
**Pull request by Iamgoofball**
Fixes the lore on the Russian Glowcaps.

### CODE REVIEW

File: code/modules/hydroponics/grown/mushrooms.dm [hunk 1]
```diff
@@ -257,13 +257,13 @@
 	product = /obj/item/weapon/reagent_containers/food/snacks/grown/mushroom/glowshroom/glowcap
 	genes = list(/datum/plant_gene/trait/glow, /datum/plant_gene/trait/cell_charge, /datum/plant_gene/trait/plant_type/fungal_metabolism)
 	mutatelist = list()
-	reagents_add = list("teslium" = 0.1, "nutriment" = 0.04)
+	reagents_add = list("teslium" = 0.1, "nutriment" = 0.04, "mindbreaker" = 0.01)
```
Comment (coiax) [inline code/modules/hydroponics/grown/mushrooms.dm]: doo dee doo undeclared change

**Comment by Cheridan** [comment]
since when are they russian

**Comment by Incoming5643** [comment]
The meme lore isn't wrong. Paramedic says they're bioluminescent but whatever the hell snake ate were generating electricity, that's not bioluminescences.

That's part of the joke.

**Comment by Incoming5643** [comment]
@Cheridan Gloomshrooms aren't Russian, Glowcaps are.

Again refer to the reference. https://www.youtube.com/watch?v=VFfHmxJbEJc

**Comment by Cheridan** [comment]
oh

closed for violating reference policy

File: code/modules/hydroponics/grown/mushrooms.dm [hunk 2]
```diff
@@ -257,13 +257,13 @@
...
 	rarity = 30
 
 /obj/item/weapon/reagent_containers/food/snacks/grown/mushroom/glowshroom/glowcap
 	seed = /obj/item/seeds/glowshroom/glowcap
 	name = "glowcap cluster"
-	desc = "<I>Mycena Ruthenia</I>: This species of mushroom glows in the dark, but aren't bioluminescent. They're warm to the touch..."
```
Comment (Supermichael777) [inline code/modules/hydroponics/grown/mushrooms.dm]: its not bioluminescent.  its radioluminescent.

**Review by Supermichael777 — CHANGES_REQUESTED** [review]

**Comment by Iamgoofball** [comment]
@Incoming5643 He was hallucinating the battery recharge, they go over this with Signit and Paramedic chatting. She confirms they're bioluminesent in the same codec call.

File: code/modules/hydroponics/grown/mushrooms.dm (hunk 1 above)
Comment (Iamgoofball) [inline code/modules/hydroponics/grown/mushrooms.dm]: part of the LORE

**Comment by Haevacht** [comment]
What is it with you and abhorrent PR titles.

**Comment by Incoming5643** [comment]
It's not a hallucination though, it actually recharges his batteries. Snake's faith in the mushrooms defied reality.

**Comment by Iamgoofball** [comment]
let's just break even and add both

**Comment by MrPerson** [comment]
Always going to downvote misleading PR titles until you get your shit together

**Comment by AnturK** [comment]
Please give this PR title that indicates the content or it will have to be closed.

**Comment by Militaires** [comment]
do they recharge your batteries

**Comment by optimumtact** [comment]
Your PR has been closed due to having a misleading or incorrect title, adjust the title and ping me to reopen it or make a new PR with a properly descriptive title and body content.

**Comment by Iamgoofball** [comment]
K

**Pull request closed without merging**